### PR TITLE
Ja yelps api [JA-1]

### PR DESCRIPTION
**Pull request by arsoljon**
* **Please check if the PR fulfills these requirements**
- [x] The pull request title contains the ticket number in the title. For example [CB-1]

* **What kind of change does this PR introduce?** (Bug fix, new feature...)
Adds code to index.js file in the JA folder.  New code gets info from the Yelp API. 

* **What is the current behavior?What does the code do before you made the new changes?** 
Current behavior gets info from Yelp api. Before this change, the code had never spoken to Yelps api. 

* **What is the new behavior?**
The new behavior gets info on some random events, specific events based on id's, and businesses near a longitude and latitude. All of which come from Yelps API. 

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR? Run a new command?)
PR introduces new communication methods to a other API. 

* **Other information**:

### CODE REVIEW

File: JA/index.js [hunk 1]
```diff
@@ -1,8 +1,55 @@
...
+            res.json({message:"unsuccessful", error: err})
+        })
+})
+
+//GET https://api.yelp.com/v3/events/{id}
+//Create a route to connect to yelps event api
+app.get('/getEvent', function(req, res){
+    const config = {headers:{"Authorization":'Bearer ' + process.env.API_TOKEN}}
+    id = req.query["id"]
+    url = 'https://api.yelp.com/v3/events/'  + id
```
Comment (jorgecan101) [inline JA/index.js]: maybe add variable types (let, const, var) to these

File: JA/index.js [hunk 2]
```diff
@@ -1,8 +1,55 @@
...
 const axios = require('axios')
+require("dotenv").config();
 app.use(express.json())
 
+
+//GET https://api.yelp.com/v3/events
+//Create a route to get the details of an event
+app.get('/getEvents', function(req, res){
+    const config = {headers:{"Authorization":'Bearer ' + process.env.API_TOKEN}}
+    url = 'https://api.yelp.com/v3/events'
```
Comment (jorgecan101) [inline JA/index.js]: maybe add variable types (let, const, var) here

**Review by Codubee — DISMISSED** [review]
LGTM!

**Review by jm220 — DISMISSED** [review]
Looks good!

**Review by jorgecan101 — DISMISSED** [review]
looks good to me

**Pull request merged**